Source/WebCore/platform/graphics/LegacyCDMSession.h

```diff
@@ -76,7 +77,7 @@ enum LegacyCDMSessionType {
     CDMSessionTypeRemote,
 };
 
-class WEBCORE_EXPORT LegacyCDMSession {
+class WEBCORE_EXPORT LegacyCDMSession : public RefCounted<LegacyCDMSession> {
```

Comment:
Since this is a virtual interface, we could have subclassed `AbstractRefCounted` instead and let the subclasses decide how they want to do ref-counting (as @geoffreygaren usually suggests).

Reply:
Deleted this code.